spec/models/scimitar/resources/base_spec.rb

```diff
@@ -4,7 +4,7 @@
   context 'basic operation' do
     FirstCustomSchema = Class.new(Scimitar::Schema::Base) do
       def self.id
-        'custom-id'
+        'urn:ietf:params:scim:schemas:custom-id'
```

Comment:
A bunch of schema ID drive-bys given recent recommendations to be mindful of the RFC IANA information re. URNs in https://datatracker.ietf.org/doc/html/rfc7643#section-10.